tests/infra_test.go

```diff
@@ -88,6 +93,24 @@ var _ = Describe("[Serial][sig-compute]Infrastructure", func() {
 		}
 	})
 
+	Describe("downwardMetrics", func() {
+		It("should be published to a vmi and periodically updated", func() {
+			vmi := libvmi.NewTestToolingFedora(
+				libvmi.WithCloudInitNoCloudUserData(tests.GetFedoraToolsGuestAgentUserData(), false),
+			)
+			tests.AddDownwardMetricsVolume(vmi, "vhostmd")
+			vmi = tests.RunVMIAndExpectLaunch(vmi, 180)
+			Expect(console.LoginToFedora(vmi)).To(Succeed())
+
+			metrics := getDownwardMetrics(vmi)
+			timestamp := getTimeFromMetrics(metrics)
```

Comment:
I think so. Basically I fetch the metrics a first time and extract the timestamp. Then I re-fetch them a few times and expect that  the timestamp has changed because the file was updated with new metrics.